apps/functions/document-check/package.json

```diff
@@ -15,6 +15,7 @@
         "@azure/functions": "^1.2.2",
         "@azure/identity": "3.0.1",
         "@pins/blob-storage-client": "^0.0.0",
+        "@pins/event-client": "^0.0.0",
```

Comment:
I'd expect a corresponding package-lock change for this.

Reply:
In the root package-lock or for this function? It doesn't look like there is a package-lock for any of the functions so far 🤔

Reply:
Root package-lock, as it's a monorepo they all use one lock file.